### PR TITLE
update release announcement for version 2.8.2

### PR DESCRIPTION
**Pull request by monikavar**
Update release announcement doc with the release of version 2.8.2 of Azure Industrial IoT platform

### CODE REVIEW

File: docs/release-announcement.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Release announcement
 
+## Azure Industrial IoT Platform Release 2.8.2
+We are pleased to announce the release of version 2.8.2 of our Industrial IoT Platform components as a second patch update of the 2.8 Long-Term Support (LTS) release. This release contains important security updates, bugfixes, backward compatibility issues with version 2.5.x and performance optimizations.
```
Comment (karok2m) [inline docs/release-announcement.md]: Can the wording for backward compatibility issues be changed a bit to say that it contains fixes for those issues ? Now it reads as if we have issues with 2.5.

Comment (monikavar) [reply docs/release-announcement.md]: done

**Review by cristipogacean — APPROVED** [review]

**Review by karok2m — APPROVED** [review]

File: docs/release-announcement.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+## Azure Industrial IoT Platform Release 2.8.2
+
+We are pleased to announce the release of version 2.8.2 of our Industrial IoT Platform components as a second patch update of the 2.8 Long-Term Support (LTS) release. This release contains important backward compatibility fixes with version 2.5.x, performance optimizations as well as security updates and bugfixes.
+
+> IMPORTANT
+> We suggest to update from  version 2.5 or later to ensure secure operations of your deployment. OPC Publisher 2.8.2 addresses backwards compatibilities issues with version 2.5.x.
+
+### Fundamentals related fixes
+
+- [OPC Publisher] Implemented the backwards compatible [Direct Methods API](docs/modules/publisher-directmethods.md) of 2.5.x publisher. The migration path is documented [here](docs/modules/publisher-migrationpath.md)
```
Comment (karok2m) [inline docs/release-announcement.md:12]: The URLs should not contain `docs/` prefix as this md file itself is already in `docs` folder.

Comment (karok2m) [reply docs/release-announcement.md:12]: Fixed in #1644

**Pull request merged**